.pre-commit-config.yaml

```diff
@@ -17,16 +17,9 @@ repos:
         entry: poetry run black
         language: system
         types: [python]
-      - id: isort
-        name: formatting::isort
-        entry: poetry run isort
-        language: system
-        types: [python]
-  - repo: local
-    hooks:
-      - id: flake8
-        name: linting::flake8
-        entry: poetry run flake8
+      - id: ruff
+        name: linting-format::ruff
+        entry: poetry run ruff
```

Comment:
Ruff reads the r`uff.toml` file and uses it as a configuration. We don't need to add a new entry in `pyproject.toml` to configure Ruff.